pkg/inline/inline.go

```diff
@@ -61,24 +77,11 @@ func (n *Inliner) InlineBundleFiles(ctx context.Context, data *bundle.Bundle) (*
 		// Because the components can themselves have file references that are
```

Comment:
It's pretty opaque to me what's going on in this whole method.  Please break it up into smaller subroutines.